metrix-mapping/src/main/groovy/com/powsybl/metrix/mapping/TimeSeriesDslLoader.groovy

```diff
@@ -371,12 +384,28 @@ class TimeSeriesDslLoader {
         binding.mapToHvdcLines = { Closure closure ->
             mapToEquipments(binding, store, config, closure, hvdcLinesFilteringContext, MappableEquipmentType.HVDC_LINE)
         }
-        binding.mapToPsts = { Closure closure ->
-            mapToPsts(binding, store, config, closure, pstFilteringContext, out)
+        binding.mapToTransformers = { Closure closure ->
+            mapToEquipments(binding, store, config, closure, transformersFilteringContext, MappableEquipmentType.TRANSFORMER)
+        }
+        binding.mapToPhaseTapChangers = { Closure closure ->
+            mapToSimpleVariableEquipments(binding, store, config, closure, phaseTapChangersFilteringContext, MappableEquipmentType.PHASE_TAP_CHANGER)
+        }
+        binding.mapToRatioTapChangers = { Closure closure ->
+            mapToSimpleVariableEquipments(binding, store, config, closure, ratioTapChangersFilteringContext, MappableEquipmentType.RATIO_TAP_CHANGER)
+        }
+        binding.mapToLccConverterStations =  { Closure closure ->
+            mapToSimpleVariableEquipments(binding, store, config, closure, lccConverterStationsFilteringContext, MappableEquipmentType.LCC_CONVERTER_STATION)
+        }
+        binding.mapToVscConverterStations =  { Closure closure ->
+            mapToSimpleVariableEquipments(binding, store, config, closure, vscConverterStationsFilteringContext, MappableEquipmentType.VSC_CONVERTER_STATION)
         }
         binding.mapToBreakers = { Closure closure ->
             mapToBreakers(binding, store, config, closure, switchesFilteringContext)
         }
+        // Kept for compatibility
```

Comment:
is @Deprecated possible here instead of comment // Kept for compatibility ?
binding.mapToPsts = { @Deprecated Closure closure